### PR TITLE
T1

### PR DESCRIPTION
**Pull request by maximKalinov**
Create application projects

### CODE REVIEW

**Review by AntsiferauGodel — APPROVED** [review]
Seems ok

**Review by HenadziBaranau — CHANGES_REQUESTED** [review]
Please also add:

- basic exception handling and logging
- Test project
- Kind of Common project for shared utilities - it could be date time providers, file operations, etc

File: NugetAnalyzer.Common/NugetAnalyzer.Common.csproj [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Http.Abstractions">
+      <HintPath>..\..\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.aspnetcore.http.abstractions\2.2.0\lib\netstandard2.0\Microsoft.AspNetCore.Http.Abstractions.dll</HintPath>
```
Comment (HenadziBaranau) [inline NugetAnalyzer.Common/NugetAnalyzer.Common.csproj]: References local system. Will get build error on build agent. Should use Nuget

**Review by HenadziBaranau — CHANGES_REQUESTED** [review]

File: NugetAnalyzer.Common/NugetAnalyzer.Common.csproj (hunk 1 above)
Comment (maximKalinov) [reply NugetAnalyzer.Common/NugetAnalyzer.Common.csproj]: Done

**Review by HenadziBaranau — APPROVED** [review]

**Review by gennadybaranov — APPROVED** [review]

**Pull request merged**